xpallets/gateway/bitcoin/v2/src/vault.rs

```diff
@@ -292,5 +293,33 @@ pub mod pallet {
                 Err(Error::<T>::VaultInactive.into())
             }
         }
+
+        /// Liquidate vault and mark it as `Liquidated`
+        ///
+        /// Liquidated vault cannot be updated.
+        pub(crate) fn liquidate_vault(id: &T::AccountId) -> Result<(), DispatchError> {
```

Comment:
https://github.com/chainx-org/ChainX/blob/e6a28871a59bf71c61fa2daeb35fa2e3eeade387/xpallets/gateway/bitcoin/v2/src/assets.rs#L138

This TODO can be partially fixed now.